README.md

```diff
@@ -62,12 +62,22 @@ Must be a list of *tensor specification keys*.
   *tensor specification keys*:
   - `name` tensor name
   - `data_type` data type (e.g. float32)
-  - `data_range` tuple of (minimum, maximum)
-  - `axes` string of axes identifying characters from: btczyx
+  - `data_range` tuple of (minimum, maximum) (e.g. [-inf, inf])
+  - `axes` string of axes identifying characters from: btczyx. Same length and order as the axes in `shape`.
   - `shape` specification of tensor shape\
-    Either as *exact shape with same length as `axes`*,\
-    or (only for input) as {`min` *minimum shape with same length as `axes`*, `step` *minimum shape change with same length as `axes`*},\
-    or (only for output) as {`reference_input` *input tensor name*, `scale` *list of factors 'output_pix/input_pix' for each dimension*, `offset` *position of origin wrt to input*, `halo` *the halo to crop from the output tensor (for example to crop away boundary efects or for tiling)*}
+    Either as *fixed shape with same length as `axes`* (e.g. `shape`: [1, 512, 512, 1]),\
+    or (only for input) as\
+    `shape`
+    - `min` *minimum shape with same length as `axes`*
+    - `step` *minimum shape change with same length as `axes`*
+    
+    or (only for output) as\
+    `shape`
+    - `reference_input` *input tensor name*
+    - `scale` *list of factors 'output_pix/input_pix' for each dimension*
+    - `offset` *position of origin wrt to input*
+    - `halo` *the halo to crop from the output tensor* (for example to crop away boundary effects or for tiling) (e.g. the valid domain of an output of `size = [256, 256]` and `halo = [32, 32]`, results in `output_shape = [256 - 2 * 32, 256 - 2 * 32] = [192, 192]`).\
+    The output shape is determined as `output_shape = reference_input_shape * scale + offset`.
```

Comment:
```suggestion
    The output shape is determined as `output_shape = reference_input_shape * scale - 2*halo.
```
in the description of `offset` it says it describes the relative position to the reference input, slighly chaning the definition of halo (really just clarifying it's meaning, see above, halo should be subtracted twice, instead

Reply:
Ok, I actually think we shouldn't include the halo in the `output_shape`, see https://github.com/bioimage-io/configuration/pull/54#discussion_r532500407.
Maybe we should make a separate issue for this discussion...